lib/uniqlogger.pro

```diff
@@ -125,8 +127,8 @@ contains ( DEFINES, ENABLE_UNQL_DBG ) {
 contains ( DEFINES, 'ENABLE_UNQL_NETLOG' ) {
     message("[*] Network Logging: ENABLED")
     QT += network
-    HEADERS +=  src/RemoteWriter.h
-    SOURCES += 	src/RemoteWriter.cpp
+    HEADERS +=  src/RemoteWriter.h src/RSyslogWriter.h
+    SOURCES += 	src/RemoteWriter.cpp src/RSyslogWriter.cpp
```

Comment:
since you're changing the line, drop the spaces so to keep the sources aligned.

Reply:
agreed, it was automatically inserted by qtcreator